packages/xml-for-vscode/src/test/suite/extension.test.ts

```diff
@@ -0,0 +1,127 @@
+import * as path from "path";
+import { getDocUri, activate, sleep } from "./helper";
+import * as vscode from "vscode";
+import { promises as fs } from "fs";
+import { expect } from "chai";
+
+const SYNTAX_ERROR_MSG = "Syntax error";
+const docPath = path.resolve(
+  __dirname,
+  "..",
+  "..",
+  "..",
+  "src",
+  "test",
+  "testFixure",
+  "test.xml"
+);
+const docUri = getDocUri(docPath);
+
+describe("Extension Test", () => {
+  before(async function() {
+    await vscode.workspace.openTextDocument(docUri);
+    await activate();
+    //wait for extension to load
+    await sleep(2000);
+  });
+  it("Test diagnostic text - lexing errors", async function() {
+    let content = `<note!>
+  <to>Tove</to>
+  <from>Jani</from>
+  <heading>Reminder</heading>
+  <body>Don't forget me this weekend!</body>
+</note>`;
```

Comment:
format the xml snippet manually so it would be better aligned with rest of JS code